docs/common/craft-parts/reference/plugins/poetry_plugin.rst

```diff
@@ -0,0 +1,105 @@
+.. _craft_parts_poetry_plugin:
+
+Poetry plugin
+=============
+
+**Version**: 2.1
+
+The Poetry plugin can be used for Python projects that use the `Poetry`_ build system.
+
+Keywords
+--------
+
+This plugin uses the common :ref:`plugin <part-properties-plugin>` keywords as
+well as those for :ref:`sources <part-properties-sources>`.
+
+Additionally, this plugin provides the plugin-specific keywords defined in the
+following sections.
+
+poetry-with:
+~~~~~~~~~~~~
+**Type:** list of strings
+
+Extra dependency groups to use other than the defaults.
+
+Environment variables
+---------------------
+
+This plugin also sets environment variables in the build environment. User-set
+environment variables will override these values. Users may also set
+`environment variables to configure poetry`_ using the
+:ref:`build-environment <build_environment>` key.
+
+PARTS_PYTHON_INTERPRETER
+~~~~~~~~~~~~~~~~~~~~~~~~
+**Default value:** python3
+
+The interpreter binary to search for in ``PATH`` or an absolute path to the
+interpreter (e.g. ``${CRAFT_STAGE}/bin/python``).
+
+PARTS_PYTHON_VENV_ARGS
+~~~~~~~~~~~~~~~~~~~~~~
+**Default value:** (empty string)
+
+Additional arguments for venv.
+
+.. _poetry-details-begin:
+
+Dependencies
+------------
+
+Python
+~~~~~~
+
+By default this plugin uses Python from the base when it is available and
```

Comment:
> Python from the base

Can we make this "the system-level Python installation"? I feel "base" is too vague.